canonical-web-and-design/ubuntu.com
# Update us pricing table to include active directory

## Done

- Update us pricing table to include active directory
- Removes ja/virtual/physical versions of the us pricing table as they are not used anywhere on the site

## QA

- Go https://ubuntu-com-11791.demos.haus/pricing/infra to and compare against [copydoc](https://docs.google.com/document/d/1Ynk0iLa9hjrdich6ATTsybtPbDwW_k4vRI0nk_HNhcI/edit#)
- Check the remove tables are not being used anywhere

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/5603
